.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": "airbnb-base",
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "semi": ["error", "never"],
+    "prefer-rest-params": 0,
+    "no-console": 0,
+    "no-underscore-dangle": 0
+  }
+};
```

Comment:
You included and configured ESLint in your project. To use it, type in terminal `eslint src` or create a script inside package.json file. It will help you greatly with keeping your code clean of typos, unused imports, etc.